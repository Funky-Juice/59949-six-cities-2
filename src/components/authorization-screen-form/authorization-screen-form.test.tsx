import * as renderer from 'react-test-renderer';
import AuthorizationScreenForm from './authorization-screen-form';

it(`AuthorizationScreenForm correctly renders`, () => {
  const tree = renderer
    .create(<AuthorizationScreenForm
      message={`test`}
      isLocked={false}
      onFormSubmit={() => {}}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
