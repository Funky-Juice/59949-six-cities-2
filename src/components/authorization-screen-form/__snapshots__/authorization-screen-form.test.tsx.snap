// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthorizationScreenForm correctly renders 1`] = `
<section
  className="login"
>
  <h1
    className="login__title"
  >
    Sign in
  </h1>
  <form
    className="login__form form"
    onSubmit={[Function]}
  >
    <div
      className="login__input-wrapper form__input-wrapper"
    >
      <label
        className="visually-hidden"
      >
        E-mail
      </label>
      <input
        className="login__input form__input"
        name="email"
        placeholder="Email"
        required={false}
        type="email"
      />
    </div>
    <div
      className="login__input-wrapper form__input-wrapper"
    >
      <label
        className="visually-hidden"
      >
        Password
      </label>
      <input
        className="login__input form__input"
        name="password"
        placeholder="Password"
        required={false}
        type="password"
      />
    </div>
    <button
      className="login__submit form__submit button"
      disabled={false}
      type="submit"
    >
      Sign in
    </button>
    <div
      style={
        Object {
          "color": "red",
          "fontSize": "14px",
          "position": "relative",
        }
      }
    >
      <p
        className="login__error-message"
        style={
          Object {
            "position": "absolute",
          }
        }
      >
        test
      </p>
    </div>
  </form>
</section>
`;
